packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -95,10 +91,27 @@ function CommentBox({
     }
   };
 
-  const handleKeydown = (e: KeyboardEvent<HTMLDivElement>) => {
-    const defaultCallback = () => onMentionKeypress(e);
-    onKeyDown(e, defaultCallback);
-    e.stopPropagation();
+  const handleKeydown = (e: KeyboardEvent<HTMLTextAreaElement>) => {
+    if ((e.ctrlKey || e.metaKey) && e.keyCode === 13 && input?.length) {
```

Comment:
What if we will we move 13 to a constant with a clear name e.g. ENTER_KEY_CODE to avoid "magic numbers"

Reply:
Sharp eyes! This was a copy-pasted code from another file, didn't realize it had some of it. Let me create a constant.